Source/WebKit/Configurations/BaseExtension.xcconfig

```diff
@@ -27,33 +27,29 @@ CLANG_ENABLE_OBJC_ARC = YES;
 CLANG_ENABLE_MODULES = YES;
 CLANG_INSTRUMENT_FOR_OPTIMIZATION_PROFILING = NO;
 INSTALL_PATH = $(WK_INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
-SWIFT_OBJC_BRIDGING_HEADER = $(SWIFT_OBJC_BRIDGING_HEADER_$(USE_INTERNAL_SDK));
-SWIFT_OBJC_BRIDGING_HEADER_ = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*] = Shared/AuxiliaryProcessExtensions/AuxiliaryProcessExtensionBridge.h;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*17.0*] = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*17.1*] = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*17.2*] = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*17.3*] = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=iphone*18.0*] = ;
-SWIFT_OBJC_BRIDGING_HEADER_YES[sdk=xr*] = ;
+SWIFT_OBJC_BRIDGING_HEADER = ;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*] = Shared/AuxiliaryProcessExtensions/AuxiliaryProcessExtensionBridge.h;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*17.0*] = ;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*17.1*] = ;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*17.2*] = ;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*17.3*] = ;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=iphone*18.0*] = ;
+SWIFT_OBJC_BRIDGING_HEADER[sdk=xr*] = ;
 SWIFT_VERSION = 5.0;
 SWIFT_OPTIMIZATION_LEVEL = -O;
 SWIFT_OPTIMIZATION_LEVEL[config=Debug] = -Onone;
 PRODUCT_NAME = $(TARGET_NAME);
 SUPPORTS_MACCATALYST = NO;
 SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD = NO;
 TARGETED_DEVICE_FAMILY = 1,2,7;
-EXCLUDED_SOURCE_FILE_NAMES = $(EXCLUDED_SOURCE_FILE_NAMES_$(USE_INTERNAL_SDK));
-EXCLUDED_SOURCE_FILE_NAMES_ = *;
-EXCLUDED_SOURCE_FILE_NAMES_YES = *;
-EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*] = ;
-EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*17.0*] = *;
-EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*17.1*] = *;
-EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*17.2*] = *;
-EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*17.3*] = *;
-EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*18.0*] = *;
-EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=xr*] = *;
+EXCLUDED_SOURCE_FILE_NAMES = *;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=iphone*] = ;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=iphone*17.0*] = *;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=iphone*17.1*] = *;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=iphone*17.2*] = *;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=iphone*17.3*] = *;
```

Comment:
...and this line here?
`EXCLUDED_SOURCE_FILE_NAMES[sdk=iphone*17.4*] = *;`

Reply:
That is a good point. For this SDK version we do want to include these source files, so there shouldn't be any excluded source files.

Thanks for reviewing!

Reply:
No problem, thanks for your reply!